### ISSUE TITLE
[ASTextNode] Is not accessible via Accessibility Inspector or UI Tests if passthroughNonlinkTouches is enabled

### ISSUE THREAD

**Issue by DavidSchechter** [facebookarchive/AsyncDisplayKit#1595]
Hello,
I have an issue while trying to make an ASTextNode accessible.
After doing both these lines:
self.textMessegeNode.accessibilityIdentifier = "textNode"
self.textMessegeNode.isAccessibilityElement = true

The text node is still not accessible.

I did the same thing for an ASImageNode like so:
self.imageMessegeNode.accessibilityIdentifier = "imageNode"
self.imageMessegeNode.isAccessibilityElement = true

And the image node is accessible.

I need the node to be accessible for UI testing, end to end testing, and for accessibility purposes.
Why is the ASTextNode not accessible, but the ASImageNode is?

Any help would be appreciated.

@appleguy I would greatly appreciate your input on this issue.

Thank you

**Comment by hannahmbanana** [comment]
@DavidSchechter: ASTextNodes _should_ be accessible. Several of the framework developers have put a great deal of effort into integrating with accessibility APIs, even in seemingly impossible situations, such as when a node hierarchy has no views associated with it (layerBacked, or completely flattened rasterized hierarchy). Even in these cases, the current version of ASDK should provide granule, specific, sub-rectangles that may each have individual accessibility parameters. 

What version of ASDK are you using? Are you on the ASDK slack channel (#1582)? This is an area of  top focus right now, so it would be great if you could be in direct contact. 

@maicki: are you aware of any issues here?

**Comment by DavidSchechter** [comment]
@hannahmbanana thank you for your quick replay.
I am currently using this commit - 402c5fe25a8bc956e8d2712ed23e89b05b7613ef - of AsyncDisplayKit.

I sent an email to join the Slack channel. Once I'm approved we can communicate there.
In the mean while, let me know if you need more information and I can post it here.

Thanks

**Comment by DavidSchechter** [comment]
Hello,
I created a sample project that can be found here  - https://github.com/DavidSchechter/testAsyncAccesbility - where the issue can be reproduced.

After talking with @maicki on the Slack channel, we figured out that the following line creates issues with accessibility for ASTextNodes -
textMessegeNode.passthroughNonlinkTouches = true

After commenting that line out, the ASTextNode is accessible.

I Hope this helps out others in the future and that this issue can be fixed.

@maicki thanks again for all you help

**Comment by appleguy** [comment]
@DavidSchechter thanks!!  We will take a look as soon as possible.  We're a few days behind on some key release stuff but should be able to circle back no later than next week.  cc @rahul-malik, @maicki

**Comment by garrettmoon** [comment]
@DavidSchechter Is this still an issue? Would you mind re uploading your test project?
